tests/fixtures/hub.py

```diff
@@ -16,9 +16,15 @@
 
 CI_HUB_ENDPOINT = "https://hub-ci.huggingface.co"
 CI_HUB_DATASETS_URL = CI_HUB_ENDPOINT + "/datasets/{repo_id}/resolve/{revision}/{path}"
+CI_HUGGINGFACE_CO_URL_TEMPLATE = CI_HUB_ENDPOINT + "/{repo_id}/resolve/{revision}/{filename}"
```

Comment:
Maybe name it CI_HFH_HUGGINGFACE_CO_URL_TEMPLATE to make it explicit that it's used in hfh ?

Reply:
I have finally removed this.